### ISSUE TITLE
TAE logo contrast is too low

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#760]
Default logo contrast is too low. User had to increase magnification from 350% to 500% to view the logo.

**Comment by cherylhjli** [comment]
@sepidehshahi Since Accessibrand did the logo, I'm not sure if I can tweak it. You may have to email Accessibrand about this issue.

**Comment by greatislander** [comment]
The logo will be responsive to all contrast themes, so I don't think we can change the default version.

**Comment by michelled** [comment]
Comments from Accessibrand: 

"No, the logo against white doesn't pass. Technically, logos don't need to pass contrast for compliance but we try to ensure they do anyways. The contrast ratios work against the dark colour to purple and blue. But to get that to work it needed to be light which fails against white.

If we want a fully contrast-accessible version for against white, we would need to either add an outline to the outer shapes or do darker versions of the colour, which would not pass against the middle shape.

In order to make the brand colours work for other design elements that's what we had to compromise on. Technically it's not a compliance issue given that logo graphics don't need it. We could slightly darken the blue and purple for the logo only so it is 3:1 against white. OR we add an outline. I think those would be your option if they want the logo fully accessible."
